umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -98,37 +112,59 @@
                          "HTTP"
                          protocol)
               operation-description (value-of service "srv:containsOperations/srv:SV_OperationMetadata/srv:operationDescription/gco:CharacterString")
-              ; [MimeType DataID DataType] [nil nil nil]]]
-              {:keys [MimeType DataID DataType]} (parse-operation-description operation-description)]]
-
+              {:keys [MimeType DataID DataType Format]} (parse-operation-description operation-description)]]
     (merge url-types
            {:URL (when url-link (url/format-url url-link sanitize?))
             :Description (char-string-value url "gmd:description")}
            (util/remove-nil-keys
-            {:GetService (when (or MimeType full-name DataID protocol DataType
+            {:GetService (when (or MimeType full-name DataID protocol DataType Format
                                   (not (empty? uris)))
                            {:MimeType (su/with-default MimeType sanitize?)
                             :FullName (su/with-default full-name sanitize?)
                             :DataID (su/with-default DataID sanitize?)
                             :Protocol (su/with-default protocol sanitize?)
                             :DataType (su/with-default DataType sanitize?)
+                            :Format (su/with-default Format sanitize?)
                             :URI (when-not (empty? uris)
                                    uris)})}))))
 
+(defn- parse-distributor-format
+  "Parses distributor format string, returns MimeTyp and Format"
+  [distributor distributor-xpaths-map]
+  (when-let [format-name (value-of distributor (get distributor-xpaths-map :Format))]
+    (let [mime-type-index (get-index-or-nil format-name "MimeType:")
+          format-index (get-index-or-nil format-name "Format:")]
+      {:Format (when format-index
+                 (let [format (subs format-name
+                                          format-index
```

Comment:
Formatting if off. Is it ironic that the word 'format' is used 4 times in this binding?